other tasks/task9.js

```diff
@@ -0,0 +1,29 @@
+(() => {
+  'use strict'
+
+  let uniqueName;
+
+  kintone.events.on(['app.record.create.submit', 'app.record.edit.submit'], (event) => {
+    //"uniqueName" = 現在のレコードの重複禁止項目の値
+    uniqueName = event.record.重複禁止項目.value;
+    console.log(uniqueName);
+
+    //"uniqueName"と重複する重複禁止項目値を持ったレコードだけを抽出
+    const params = {
+      'fields': '重複禁止項目',
+      'app': 31,
+      'query': '重複禁止項目 = "' + uniqueName + '"'
```

Comment:
```suggestion
      'query': `重複禁止項目 = "${uniqueName}" `
```

クエリーの値はシングルクォートとダブルクォートが混じってしまい読みにくくなっているので、テンプレートリテラルを使って書き換えてみましょう！

また、現状のクエリーだと、
編集画面で何も変更せずに保存したときに「レコードが重複しています。」のアラートが出てしまいます。
そのため、レコード編集時は現在編集中のレコードを含めないようなクエリーにする必要があります！
つまり、レコード新規追加時と編集時でクエリーをわける条件分岐が必要です！